apps/beeai-server/src/beeai_server/configuration.py

```diff
@@ -128,13 +128,21 @@ class Configuration(BaseSettings):
     platform_service_url: str = "beeai-platform-svc:8333"
     port: int = 8333
 
+    base_path: str = "/"
+
     @model_validator(mode="after")
     def _oci_registry_defaultdict(self):
         oci_registry = defaultdict(OCIRegistryConfiguration)
         oci_registry.update(self.oci_registry)
         self.oci_registry = oci_registry
         return self
 
+    @model_validator(mode="after")
+    def _base_path_ends_in_slash(self):
+        if not self.base_path.endswith("/"):
+            self.base_path += "/"
+        return self
```

Comment:
This is a bit cumbersome. I think you could do this more elegantly by either joining using `os.path.join` (which handles `/` automatically) or by defining the field like `Field(default="/", pattern=r".*/$")`.